xacro/__init__.py

```diff
@@ -171,41 +181,58 @@ def check_deprecated_tag(tag_name):
 
 
 class Macro(object):
+
     def __init__(self):
         self.body = None  # original xml.dom.Node
         self.params = []  # parsed parameter names
         self.defaultmap = {}  # default parameter values
         self.history = []  # definition history
 
 
+class ResourceNotFound(Exception):
+    pass
+
+
 def eval_extension(s):
-    if s == '$(cwd)':
+    # TODO This is Temporary fix to return cwd() for 'find xacro' command.
+    # We can use resolve_args() when rospkg will be migrated to ROS2.
+    if s == '$(cwd)' or s == '$(find xacro)':
```

Comment:
I guess this temporary fix isn't necessary since we're using ament to find packages now in [substitution_args.py](https://github.com/ros/xacro/blob/680c147092e76a7e1acfa3d42cd67c7e835850eb/xacro/substitution_args.py#L34).